cli/src/complete.rs

```diff
@@ -409,20 +410,23 @@ fn config_keys_rec(
                     continue;
                 };
                 let properties = properties.as_object().unwrap();
-                config_keys_rec(prefix, properties, acc, only_leaves);
+                config_keys_rec(prefix, properties, acc, only_leaves, include_eq);
             }
             _ => {
                 let help = value
                     .get("description")
                     .map(|desc| desc.as_str().unwrap().to_string().into());
-                let escaped_key = prefix.to_string();
+                let mut escaped_key = prefix.to_string();
+                if include_eq {
+                    escaped_key.push('=');
+                }
                 acc.push(CompletionCandidate::new(escaped_key).help(help));
             }
         }
     }
 }
 
-fn config_keys_impl(only_leaves: bool) -> Vec<CompletionCandidate> {
+fn config_keys_impl(only_leaves: bool, include_eq: bool) -> Vec<CompletionCandidate> {
```

Comment:
nit: (at your option) `suffix: &str` could be added instead (to avoid `(bool, bool)`), but that's not important. One minor good thing is that we can use `format!("{name}{suffix}")` to construct an escaped key.

Reply:
good idea